### DISCUSSION TITLE
Integrate ValueSERP API

### DISCUSSION THREAD

**Discussion by jagdish1o1** [towfiqi/serpbear#106] [Ideas]
I love the design and everything about this tool. The only thing is lacking is a good and cheap api integration. 

Currently the free options have issues and not accurate too, I guess we cannot except much from free options. 

I personally use ValueSERP alot and it's a good and cheapest available in the market. 

Please, Integrate valueserp api if possible. 

Thanks.

**Comment by towfiqi** [comment]
ValueSerpt Integrated in v1.0.0
